examples/bootstrap_server/bootstrap_server.c

```diff
@@ -617,6 +619,19 @@ int main(int argc, char *argv[])
         case '4':
             data.addressFamily = AF_INET;
             break;
+        case 'S':
+            opt++;
+            if (opt >= argc) {
+                print_usage(filename, port);
+                return 0;
+            }
+            uint16_t coap_block_size_arg;
+            if (1 == sscanf(argv[opt], "%hu", &coap_block_size_arg) && lwm2m_set_block_size(coap_block_size_arg)) {
+                break;
+            } else {
+                print_usage(filename, port);
+                return 0;
+            }
```

Comment:
The else part could be removed, moving code more to the left:
```suggestion
            if (!(1 == sscanf(argv[opt], "%hu", &coap_block_size_arg) && lwm2m_set_block_size(coap_block_size_arg))) {
                print_usage(filename, port);
                return 0;
            }
            break;
```
(opinionated, feel free to ignore)